src/org/traccar/protocol/MiniFinderProtocolDecoder.java

```diff
@@ -34,44 +34,145 @@ public MiniFinderProtocolDecoder(MiniFinderProtocol protocol) {
         super(protocol);
     }
 
-    private static final Pattern PATTERN = new PatternBuilder()
-            .expression("![A-D],")
+    // The !A record is not in the documentation for MiniFinder or EV-07S.
```

Comment:
I think we should remove these comments. You can put them into the pull request description, so we can always go back and check why it was done like this, but I don't think we need to keep them in the code.